Buildings/Controls/OBC/Utilities/PIDWithAutotuning/AutoTuner/AMIGO/PIDIntegralTime.mo

```diff
@@ -0,0 +1,100 @@
+﻿within Buildings.Controls.OBC.Utilities.PIDWithAutotuning.AutoTuner.AMIGO;
+block PIDIntegralTime "Identifies the integral time of a PID controller"
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput T(final min=0)
+    "Time constant of a first order time-delayed model"
+    annotation (Placement(transformation(extent={{-140,40},{-100,80}}),
+        iconTransformation(extent={{-140,40},{-100,80}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput L(final min=1E-6)
+    "Time delay of a first order time-delayed model"
+    annotation (Placement(transformation(extent={{-140,-80},{-100,-40}}),
+        iconTransformation(extent={{-140,-80},{-100,-40}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput Ti
+    "Time constant signal for the integral term"
+    annotation (Placement(transformation(extent={{100,-20},{140,20}})));
+  Buildings.Controls.OBC.CDL.Continuous.Add add1
+    "Calculate the sum of the time delay and the output of gai3"
```

Comment:
I would avoid to cite any instance name in the comment, like in this one, there is `gai3`. It may not be correct if in the future we change the instance name `gai3` to something else.

Reply:
done